R/get_current_measurements_fast.R

```diff
@@ -0,0 +1,31 @@
+#' Get current measurements of traffic
+#'
+#' @export
+#'
+#' @examples
+#' # Put example code here
+get_current_measurements_fast<- function() {
+  url <- "http://opendata.ndw.nu/measurement_current.xml.gz"
+  cachedir <- rappdirs::user_cache_dir(appname = "informalr")
+  destdir <- file.path(cachedir, "extdata")
```

Comment:
I see no added value for `destdir`, that is, why not use `cachedir`. Is there any? If yes, it's brilliant. If not, I'll create a 'Good First Issue' :=1: